staging/kos/pkg/cmd/server/start.go

```diff
@@ -49,9 +55,8 @@ func NewNetworkAPIServerOptions(out, errOut io.Writer) *NetworkAPIServerOptions
 	o := &NetworkAPIServerOptions{
 		RecommendedOptions: genericoptions.NewRecommendedOptions(defaultEtcdPathPrefix, apiserver.Codecs.LegacyCodec(v1alpha1.SchemeGroupVersion)),
 		ServerRunOptions:   genericoptions.NewServerRunOptions(),
-
-		StdOut: out,
-		StdErr: errOut,
+		StdOut:             out,
```

Comment:
I think this function is returning _recommended_ options, and so should set `CheckSubnetsConflicts = true`.

Reply:
Fixed in 8ac9883.